Tests/unit/PolarisServer.Tests.ps1

```diff
@@ -19,26 +19,23 @@ Describe "Test webserver use" {
     }
 
     Context "Test starting and stopping of the server" {
-        BeforeAll {
-            if (-not $IsUnix) {
-                Stop-Polaris
-                $Polaris = Get-Polaris
-                $Polaris.Listener.IsListening | Should Be $false
-
-                $Polaris = Start-Polaris -Port 9998
-                $Polaris.Listener.IsListening | Should be $true
-            }
+
+        It "Should allow starting and stopping the server" {
```

Comment:
I think we can probably just have one tests that stops/starts/stop/starts Polaris. It's typically best practice that single tests don't depend on something happening in another test.

Reply:
I think I got your meaning. How does it look now?

Reply:
Yes that looks good! Just wondering though... any reason you removed the "Allow a custom logger" test?

Reply:
The exact reason would be that I am a dingbat... haha. Added the test back in, thanks for catching that.